exporters/sender/jdk/src/main/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSender.java

```diff
@@ -221,7 +221,13 @@ private HttpResponse<byte[]> sendRequest(
   }
 
   private static boolean isRetryableException(IOException throwable) {
-    return throwable instanceof HttpTimeoutException;
+    // Almost all IOExceptions we've encountered are transient retryable, so we opt out of specific
+    // IOExceptions that are unlikely to resolve rather than opting in.
+    // Known retryable IOException messages: "Connection reset", "/{remote ip}:{remote port} GOAWAY
+    // received"
+    // Known retryable HttpTimeoutException messages: "request timed out"
+    // Known retryable HttpConnectTimeoutException messages: "HTTP connect timed out"
+    return !(throwable instanceof SSLException);
```

Comment:
should https://github.com/open-telemetry/opentelemetry-java/blob/1ecc9197bea8a566bf4ec6e3afa6b70f55d593ef/exporters/sender/okhttp/src/main/java/io/opentelemetry/exporter/sender/okhttp/internal/RetryInterceptor.java#L102-L110
be also modified?

Reply:
I'm not sure the behavior of the JDK HttpClient is applicable to OkHttp client.